test/e2e-tests-latest-serving.sh

```diff
@@ -0,0 +1,85 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script runs the end-to-end tests against Knative Serving
+# Operator built from source. However, this script will download the latest
+# source code of knative serving and generated the latest manifest file for
+# serving installation. The serving operator will use this newly generated
+# manifest file to replace the one under the directory
+# cmd/manager/kodata/knative-serving. This purpose of this script is to verify
+# whether the latest source code of operator can work properly with the latest
+# source code of knative serving.
+
+# If you already have a Knative cluster setup and kubectl pointing
+# to it, call this script with the --run-tests arguments and it will use
+# the cluster and run the tests.
+
+# Calling this script without arguments will create a new cluster in
+# project $PROJECT_ID, start knative in it, run the tests and delete the
+# cluster.
+
+source $(dirname $0)/e2e-common.sh
+
+OPERATOR_DIR=$(dirname $0)/..
+KNATIVE_SERVING_DIR=$OPERATOR_DIR/..
```

Comment:
can you enclose the variable with { }, as in 
KNATIVE_SERVING_DIR=${OPERATOR_DIR}/..

This will make it easier to read what the var is, same for the rest of the file.

Reply:
OK